cfme/infrastructure/pxe.py

```diff
@@ -630,7 +670,7 @@ class PXESystemImageTypeDetailsView(PXEMainView):
 
     @View.nested
     class entities(View):  # noqa
-        basic_information = SummaryTable(title="Basic Information")
+        basic_information = Table(title="Basic Information")
```

Comment:
**Required**

I don't see any updates to this widget use to go along with this change and it looks like a SummaryTable to me in the UI, what's the reason for this change?

Reply:
I did make use of it, but then changed the code but forgot to change the widget.